include/qrack_defines.hpp.in

```diff
@@ -0,0 +1,4 @@
+#cmakedefine ENABLE_OPENCL 1
+#cmakedefine ENABLE_COMPLEX_X2 1
+#cmakedefine ENABLE_COMPLEX8 1
```

Comment:
Have you tested the result of this in both directions? Specifically that these are defined to 0 when not set?

Reply:
Yes, I just did.